src/accessories/HandHelper.ttslua

```diff
@@ -37,6 +52,55 @@ function updateColors()
   self.setName(handColor .. " Hand Helper")
 end
 
+function getHandSizeModifier()
+  local modSize = 0
+  local deckAreaObjects = playermatApi.getDeckAreaObjects(matColor)
+
+  -- look for  cards around playermat that modify hand size
+  for _, obj in ipairs(playermatApi.searchAroundPlayermat(matColor, "isCard")) do
+    if not obj.is_face_down and
+      obj ~= deckAreaObjects.draw and
+      obj ~= deckAreaObjects.topCard and
+      obj ~= deckAreaObjects.discard then
+
+      -- handle specific cards
+      local md = JSON.decode(obj.getGMNotes()) or {}
+      if md.id == "60205" then -- Arcane Enlightenment
+        modSize = modSize + 1
+      elseif md.id == "02020" then -- Laboratory Assistant
+        modSize = modSize + 2
+      elseif md.id == "60202" then -- Vault of Knowledge
+        modSize = modSize + 2
+      elseif md.id == "03041" then -- Drawing the Sign
+        modSize = modSize - 5
+      elseif md.id == "71023" then -- Dr. Mya Badry
+        modSize = modSize + 2
```

Comment:
You could make this into a global table:
```lua
local handModifiers = {
  "60205" = 1,
  -- etc.
  }
  ```
  And then just do `modSize = modSize + (handModifiers[md.id] or 0)`